src/sentry/lang/javascript/processor.py

```diff
@@ -219,6 +221,10 @@ def get_release_file_cache_key_meta(release_id, releasefile_ident):
     return "meta:%s" % get_release_file_cache_key(release_id, releasefile_ident)
 
 
+def should_retry_fetch(attempt: int, e: Exception) -> bool:
+    return not attempt > 3 and isinstance(e, OSError) and e.errno == errno.ESTALE
```

Comment:
No specific reason to choose 3 over any other number.

Reply:
It could make sense to introduce a constant for this at least to make it more discoverable.

For a moment, I was thinking if the attempts should be controlled by the retry policy rather than this function. However, this allows you to create time-based retries, too. So 👍  for handling this in here.